### ISSUE TITLE
Too much "Set Destination" everywhere.

### ISSUE THREAD

**Issue by flukeout** [mozilla/webmaker-core#521]
When I'm picking a destination for the button, I see this page...

![image](https://cloud.githubusercontent.com/assets/25212/9238273/f24d30e4-4105-11e5-96bf-8961d7299ea7.png)

I think the button should say something like "OK" or "Done" and not "Set Destination", which the header already says. Maybe we need a better header here anyway, something like "Pick a page for this button to link to".

@xmatthewx @vazquez Thoughts?

**Comment by xmatthewx** [comment]
You're right. 'OK' works for me. 'Pick destination' up top would also more user-friendly than 'Set destination' for non-coders.
